besu/src/main/java/org/hyperledger/besu/RunnerBuilder.java

```diff
@@ -1469,9 +1468,8 @@ private WebSocketService createWebsocketService(
         vertx, configuration, websocketMessageHandler, authenticationService, metricsSystem);
   }
 
-  private Optional<MetricsService> createMetricsService(
-      final Vertx vertx, final MetricsConfiguration configuration) {
-    return MetricsService.create(vertx, configuration, metricsSystem);
+  private Optional<MetricsService> createMetricsService(final MetricsConfiguration configuration) {
+    return MetricsService.create(configuration, metricsSystem);
```

Comment:
Vertx was used to create our own custom implementation of the Prometheus HTTP server, to avoid reinventing the wheel, I just use the HTTP server provided by the Prometheus lib, that has also newer bring new feature without any effort from our side, for this reason Vertx is not needed anymore.